docs/source/installation/mac.rst

```diff
@@ -42,8 +42,14 @@ To install LaTeX:
 
    brew install --cask basictex
 
-Ensure that the TeXLiveManager tool ``tlmgr`` works, by running ``tlmgr
---version`` in the terminal.  Now run
+Reboot your computer then ensure that the TeXLiveManager tool ``tlmgr`` works by running ``tlmgr
+--version`` in the terminal. Now update ``tlmgr``
```

Comment:
Indeed, I'd be a bit more comfortable with a weaker formulation:

```suggestion
Ensure that the TeXLiveManager tool ``tlmgr`` works by running ``tlmgr
--version`` in the terminal (a reboot might be necessary). Now update ``tlmgr``
```

(But if you disagree, I won't block this PR over it.)

Reply:
Agreed